signals.go

```diff
@@ -114,7 +114,7 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 		default:
 			us := s.(unix.Signal)
 			logrus.Debugf("forwarding signal %d (%s) to %d", int(us), unix.SignalName(us), pid1)
-			if err := unix.Kill(pid1, us); err != nil {
+			if err := process.Signal(s); err != nil {
```

Comment:
Sorry, the above comments should be in here.

Reply:
Well, `os.Signal` documentation says `on Unix it is syscall.Signal`. This is why we do `us := s.(unix.Signal)` without checking for errors. I doubt this will ever change.

Reply:
> We didn't check this transform result in before, if failed, we will also send a null signal to the container.

Nope, `us := s.(unix.Signal)` will panic at runtime if the type assertion fails.

Reply:
Ah, I forgot this.